.cirrus.yml

```diff
@@ -0,0 +1,95 @@
+env:
+  # Determine what we need here
+
+eks_container: &BUILD_TEMPLATE
+  dockerfile: .cirrus/Dockerfile
+  builder_subnet_id: ${CIRRUS_AWS_SUBNET}
+  builder_role: cirrus-builder
+  builder_image: docker-builder-v*
+  builder_instance_type: t2.small
+  cluster_name: ${CIRRUS_CLUSTER_NAME}
+  region: eu-central-1
+  namespace: default
+  docker_arguments:
+    CIRRUS_AWS_ACCOUNT: ${CIRRUS_AWS_ACCOUNT}
+
+yarn_cache_template: &YARN_CACHE_TEMPLATE
+  yarn_cache:
+    folders:
+      - '.yarn'
+    fingerprint_script: cat yarn.lock
```

Comment:
so we are going full-on pnp to try with zero install and commit the dependencies? that means no need for this task and the install_dependencies_task?

Reply:
So yes you're right, zero-install includes them
https://github.com/yarnpkg/berry/issues/454#issuecomment-530312089

I looked at yarn itself
https://github.com/yarnpkg/berry/tree/master

it has commits the .yarn/cache and the pnp files. So lets do it!

Reply:
second though will do it as a new PR

Reply:
yeah make sense!